app/devices/home.py

```diff
@@ -143,34 +171,57 @@ def self_sufficiency(self) -> float:
         self._solar_production_power = assign_if_available(
-            self._solar_production_power, state_repository.get_state(self._solar_power_entity_id))
+            self._solar_production_power,
+            state_repository.get_state(self._solar_power_entity_id),
+        )
         self._grid_imported_power = assign_if_available(
-            self._grid_imported_power, state_repository.get_state(self._grid_supply_power_entity_id))
+            self._grid_imported_power,
+            state_repository.get_state(self._grid_supply_power_entity_id),
+        )
 
         self._produced_solar_energy = assign_if_available(
-            self._produced_solar_energy, state_repository.get_state(self._solar_energy_entity_id))
+            self._produced_solar_energy,
+            state_repository.get_state(self._solar_energy_entity_id),
+        )
         self._grid_imported_energy = assign_if_available(
-            self._grid_imported_energy, state_repository.get_state(self._grid_imported_energy_entity_id))
+            self._grid_imported_energy,
+            state_repository.get_state(self._grid_imported_energy_entity_id),
+        )
         self._grid_exported_energy = assign_if_available(
-            self._grid_exported_energy, state_repository.get_state(self._grid_exported_energy_entity_id))
-
-        self._consumed_energy = self.grid_imported_energy - \
-            self.grid_exported_energy + self.produced_solar_energy
-        self._consumed_solar_energy = self.produced_solar_energy - self.grid_exported_energy
+            self._grid_exported_energy,
+            state_repository.get_state(self._grid_exported_energy_entity_id),
+        )
+
+        self._consumed_energy = (
+            self.grid_imported_energy
+            - self.grid_exported_energy
+            + self.produced_solar_energy
+        )
+        self._consumed_solar_energy = (
+            self.produced_solar_energy - self.grid_exported_energy
+        )
 
         if self._energy_snapshop is None:
-            self.set_snapshot(self.consumed_solar_energy, self.consumed_energy,
-                              self.produced_solar_energy, self.grid_imported_energy, self.grid_exported_energy)
+            self.set_snapshot(
+                self.consumed_solar_energy,
+                self.consumed_energy,
+                self.produced_solar_energy,
+                self.grid_imported_energy,
+                self.grid_exported_energy,
+            )
 
         for device in self.devices:
             await device.update_state(state_repository, self.self_sufficiency)
 
-    async def update_power_consumption(self, state_repository: StatesRepository, optimizer: Optimizer) -> None:
-        """"Update the device based on the current pv availablity."""
+    async def update_power_consumption(
+        self, state_repository: StatesRepository, optimizer: Optimizer
+    ) -> None:
+        """ "Update the device based on the current pv availablity."""
```

Comment:
Please remove this warning:

app/devices/home.py:219:9: D210 No whitespaces allowed surrounding docstring text